cuit-service.sln

```diff
@@ -18,7 +18,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		build-n-publish.sh = build-n-publish.sh
 		demo.http = demo.http
 		Dockerfile = Dockerfile
-		global.json = global.json
```

Comment:
If I am not wrong, deleting `global.json` file is missing.

Reply:
I need only need modify the content of the global.json, not delete it! haha

Reply:
No, no, you need to delete it!